lib/rules/await-async-queries.ts

```diff
@@ -75,22 +78,39 @@ export default createTestingLibraryRule<Options, MessageIds>({
 						closestCallExpressionNode.parent
 					);
 
-					// check direct usage of async query:
-					// const element = await findByRole('button')
+					/**
+					 * Check direct usage of async query:
+					 * const element = await findByRole('button');
+					 */
 					if (references.length === 0) {
 						if (!isPromiseHandled(identifierNode)) {
 							context.report({
 								node: identifierNode,
 								messageId: 'awaitAsyncQuery',
 								data: { name: identifierNode.name },
+								fix: (fixer) => {
+									if (
+										isMemberExpression(identifierNode.parent) &&
+										ASTUtils.isIdentifier(identifierNode.parent.object) &&
+										identifierNode.parent.object.name === 'screen'
```

Comment:
Does it really matter if the parent object is `screen`? I think it should work no matter what the name of its parent is.

For example:

```js
test("An example test", async () => {
  const view = render(<Component />)
  const button = view.findByText('submit')
});
```

In this case, `view.findByText` wouldn't be autofixed, but it should. Can you add this scenario to the tests, and adjust the rule fix accordingly?